grow/templates/jinja_dependency.py

```diff
@@ -0,0 +1,112 @@
+"""Track the jinja2 rendering dependencies."""
```

Comment:
What does a developer need to know about the maintainability of `DepEnvironment`? Is there any way to simply have a loader to track the deps instead of having a custom Environment? It might be good to have a quick comment in the file about how to update this or why it was necessary. :)